proto/01-protoc-base/requirements.txt

```diff
@@ -1,2 +1,3 @@
-grpcio-tools==1.63.0
-mypy-protobuf>=3.6.0
+grpcio-tools==1.76.0
+mypy-protobuf==4.0.0
```

Comment:
Mentioned in DM, but ideally we would not lock down versions that don't "need" to be locked down; we get the benefit of locking from the docker image itself. If we encounter an error downstream, we can choose to fix the linting at that time, or temporarily lock the version and file an issue to fix the issue later.